features/user_coach_auth.feature

```diff
@@ -177,15 +156,8 @@ Scenario: Sad Path - Cannot create new iteration, edit and destroy existing iter
     | id | name  | github_uid      | email          | user_type     |
     | 1  | user1 | esaas_developer | test@user.com  | student       |
   And I am logged in
-  And I follow "Apps"
-  And I follow "app1"
-  And I follow "2017-03-25"
-  And I follow "Add Iteration"
-  Then I should see "Error: Only Staff can create, edit and destroy apps, engagements and iterations"
-  And I press "Edit" for "2017-04-14"
-  Then I should see "Error: Only Staff can create, edit and destroy apps, engagements and iterations"
-  And I press "Destroy" for "2017-04-14"
-  Then I should see "Error: Only Staff can create, edit and destroy apps, engagements and iterations"
+  And I am on the Apps page
+  Then I should see "You do not have access to that page."
```

Comment:
Already included in previous PR, but to pass CI, we have to include modified tests.

Please ignore this file.